Sources/Bot/Services/MergeService.swift

```diff
@@ -111,13 +111,13 @@ extension MergeService {
                         case 0:
                             return github.postComment(
                                 "Your pull request was accepted and is going to be handled right away 🏎",
```

Comment:
```suggestion
                                "Your pull request was accepted and is going to be handled very soon 🏎",
```
Just wording, but since we saw some race conditions in the past in Steve, where 2 PRs were having that message posted in the same time but one of them were in fact waiting in the queue...

Reply:
This is going to be handled separately.

Reply:
managing expectations FTW

Reply:
@AliSoftware @ilyapuchka fixed in https://github.com/Babylonpartners/Wall-E/pull/21.